.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think these files might be particular to your local comp? maybe you can add the .vs to the gitignore file?

Reply:
I think the same goes for some other files in here, we can look at that later.

Reply:
Got it!